### PR TITLE
Fix Infrastructure component locators for 6.10

### PR DESCRIPTION
**Pull request by jameerpathan111**
Some of locators for landing page of Satellite 6.10 components have changed because of PF4 change. This PR fixes the landing page locators for those pages.

### CODE REVIEW

File: airgun/views/computeresource.py [hunk 1]
```diff
@@ -24,8 +24,8 @@
 
 class ComputeResourcesView(BaseLoggedInView, SearchableViewMixin):
 
-    title = Text("//h1[text()='Compute Resources']")
-    new = Text("//a[contains(@href, '/compute_resources/new')]")
+    title = Text('//h5[text()="Compute Resources"]')
```
[the remark this answers is not in the document]
Comment (jameerpathan111) [reply airgun/views/computeresource.py:27]: @SatelliteQE/airgun-tier-1-reviewers @SatelliteQE/airgun-tier-2-reviewers anyone knows how we can fix this? or we should ignore it for now? Cause we don't really use/need `title` anywhere.

[the remark this answers is not in the document]
Comment (mshriver) [reply airgun/views/computeresource.py:27]: What do you mean uses h1 and h5?  Across Sat versions, or in 6.10 the locator could be either?

xpath supports selecting multiple element types dynamically via a syntax like:
```//*[(self::div or self::a) and text()="Compute Resources"]```

[the remark this answers is not in the document]
Comment (jameerpathan111) [reply airgun/views/computeresource.py]: > What do you mean uses h1 and h5? Across Sat versions, or in 6.10 the locator could be either?

When there is no entity created `title` locator is `h5` but once we create entities we no longer see the original page, and the new page's title locator is to `h1`. In satellite 6.9 both locator were `h1`.
> xpath supports selecting multiple element types dynamically via a syntax like:
> `//*[(self::h1 or self::h5) and text()="Compute Resources"]`

Thanks, this worked, updated the pr. :)

**Review by rplevka — APPROVED** [review]

**Comment by jameerpathan111** [comment]
I had accidentally added an empty file in pr so removed it, and force pushed changes.

**Review by jyejare — APPROVED** [review]

File: airgun/views/computeprofile.py [hunk 2]
```diff
@@ -9,8 +9,8 @@
 
 
 class ComputeProfilesView(BaseLoggedInView, SearchableViewMixin):
-    title = Text("//h1[text()='Compute Profiles']")
-    new = Text("//a[contains(@href, '/compute_profiles/new')]")
+    title = Text('//*[(self::h1 or self::h5) and text()="Compute Profiles"]')
```
Comment (latran) [inline airgun/views/computeprofile.py:12]: I can't find where the h5 is coming from for changes like this.  I only see h1.

Comment (latran) [reply airgun/views/computeprofile.py:12]: nvm, I see where they're from now.

**Review by latran — CHANGES_REQUESTED** [review]
Ok LGTM. Pending test results.

**Comment by jameerpathan111** [comment]
Test result:

```
$ pytest tests/foreman/ui/test_computeprofiles.py 
============================= test session starts ==============================
platform linux -- Python 3.8.6, pytest-6.2.2, py-1.10.0, pluggy-0.13.1
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/jpathan/projects/robottelo, configfile: pyproject.toml
plugins: ibutsu-1.13, services-2.2.1, cov-2.11.1, mock-3.5.1, reportportal-5.0.7, xdist-2.2.1, forked-1.3.0
collected 1 item

tests/foreman/ui/test_computeprofiles.py .                               [100%]

================== 1 passed, 13 warnings in 111.71s (0:01:51) ==================

```
```
$ pytest tests/foreman/ui/test_subnet.py -k test_positive_end_to_end 
============================= test session starts ==============================
platform linux -- Python 3.8.6, pytest-6.2.2, py-1.10.0, pluggy-0.13.1
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/jpathan/projects/robottelo, configfile: pyproject.toml
plugins: ibutsu-1.13, services-2.2.1, cov-2.11.1, mock-3.5.1, reportportal-5.0.7, xdist-2.2.1, forked-1.3.0
collected 1 item

tests/foreman/ui/test_subnet.py .                                        [100%]

================== 1 passed, 17 warnings in 156.88s (0:02:36) ==================

```
```
$ pytest tests/foreman/ui/test_domain.py -k test_positive_end_to_end 
============================= test session starts ==============================
platform linux -- Python 3.8.6, pytest-6.2.2, py-1.10.0, pluggy-0.13.1
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/jpathan/projects/robottelo, configfile: pyproject.toml
plugins: ibutsu-1.13, services-2.2.1, cov-2.11.1, mock-3.5.1, reportportal-5.0.7, xdist-2.2.1, forked-1.3.0
collected 6 items / 5 deselected / 1 selected

tests/foreman/ui/test_domain.py .                                        [100%]

=========== 1 passed, 5 deselected, 10 warnings in 176.66s (0:02:56) ===========
```
Can't provide test result for http proxy test because of [bz#1966942](https://bugzilla.redhat.com/show_bug.cgi?id=1966942), but the changes are irrelevant to issue and works fine for me.
Note: This pr depends on https://github.com/SatelliteQE/robottelo/pull/8664

**Review by latran — APPROVED** [review]
ACK

**Review by mshriver — APPROVED** [review]

**Pull request merged**